utils/json-schemas/device_registration.json

```diff
@@ -0,0 +1,76 @@
+{
+  "$schema": "https://frinx.io",
```

Comment:
this should be definition of JSON schema itself based on https://json-schema.org/draft/2020-12/release-notes

`https://json-schema.org/draft/2020-12/schema`